assets/app/scripts/services/buildConfigs.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("BuildConfigsService", function(){
```

Comment:
> So shall I put them back ?

I would :-/

The functions only make sense inside the controller, so I would keep them there.